src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -46,37 +48,37 @@ public async Task InitializeAsync(ITextView textView, SuggestionServiceBase sugg
         public async Task GenerateDocumentationProposalAsync(DocumentationCommentSnippet snippet,
             ITextSnapshot oldSnapshot, VirtualSnapshotPoint oldCaret, CancellationToken cancellationToken)
         {
-            await Task.Yield().ConfigureAwait(false);
+            await TaskScheduler.Default;
```

Comment:
Maybe wait until _after_ the `!Enabled` check before yielding the thread to your caller, in case you don't need to yield at all.